src/objectid.ts

```diff
@@ -1,6 +1,10 @@
 import { Buffer } from 'buffer';
-import { deprecate, inspect } from 'util';
-import { haveBuffer, randomBytes } from './parser/utils';
+import { ensureBuffer } from './ensure_buffer';
+import { BufferEncoding, haveBuffer, randomBytes } from './parser/utils';
+
+declare const require: Function;
```

Comment:
is this needed if you disable `no-var-requires` below? It also occurs to me that `util` requires us to bring in node support for the web bundle, maybe we should consider replacing `deprecate` and only use `inspect` when used server-side

Reply:
This is needed because I removed node types so this just a little patch to allow use to use require in the "is this node" guarded code. 

NODE-2826